core/txfeed/txfeed.go

```diff
@@ -130,6 +130,9 @@ func (t *Tracker) Find(ctx context.Context, id, alias string) (*TxFeed, error) {
 	)
 
 	err := t.DB.QueryRow(ctx, q.String(), id).Scan(&feed.ID, &sqlAlias, &feed.Filter, &feed.After)
+	if err == sql.ErrNoRows {
+		return nil, errors.WithDetailf(pg.ErrUserInputNotFound, "alias: %s", alias)
```

Comment:
We have a new function! https://godoc.org/github.com/chain/chain/errors#Sub

I suggest:

```
err = errors.Sub(pg.ErrUserInputNotFound, err)
err = errors.WithDetailf(err, "alias: %s", alias)
return nil, err
```

This way if err already has a stack trace or detail or whatever, they'll all be preserved.